src/foam/nanos/auth/User.js

```diff
@@ -99,16 +99,7 @@ foam.CLASS({
     {
       class: 'String',
       name: 'firstName',
-      documentation: 'The first name of the User.',
-      validateObj: function(firstName) {
-        if ( ! firstName.trim() ){
-          return 'First Name Required.'
-        } if ( firstName.length > 70 ) {
-          return 'First name cannot exceed 70 characters.';
-        } if( /\d/.test(this.firstName) ) {
-          return 'First name cannot contain numbers';
-        }
-      }
+      documentation: 'The first name of the User.'
```

Comment:
Don't you want a minLength here and below or are you ok with these fields being empty strings?

Reply:
as default on the model, we are ok at least right now with empty string. Setting defaults here breaks things across the board. 

For example a contact is a user that does not need firstName

Reply:
We did talk about this in person, but considering this involves making changes to **Contact** to ensure no requirements, and the fact that some places are using the **User** incorrectly. I'm going to open a ticket for this. And do this separately.

Reply:
ticket: https://nanopay.atlassian.net/browse/CPF-2461